src/main/java/controller/BlackJackGame.java

```diff
@@ -0,0 +1,58 @@
+package controller;
+
+import java.util.List;
+
+import domain.result.WinningResult;
+import domain.deck.Deck;
+import domain.deck.DeckFactory;
+import domain.user.Dealer;
+import domain.user.Player;
+import domain.user.Players;
+import util.YesOrNo;
+import view.InputView;
+import view.OutputView;
+
+public class BlackJackGame {
+
+    private static final int FIRST_CARD_COUNT = 2;
+    private final Deck deck;
+
+    public BlackJackGame() {
+        deck = DeckFactory.getDeck();
+    }
+
+    public void firstDealOut(Dealer dealer, Players players) {
+        for (int i = 0; i < FIRST_CARD_COUNT; i++) {
+            dealer.draw(deck.dealOut());
+            players.draw(deck);
+        }
+    }
+
+    public void additionalDealOut(Dealer dealer, Players players) {
+        players.getPlayers()
+                .forEach(this::playersAdditionalDraw);
+
+        while (dealer.isAvailableToDraw()) {
+            OutputView.printDealerDealOut();
+            dealer.draw(deck.dealOut());
+        }
+    }
+
+    private void playersAdditionalDraw(Player player) {
+        while (player.isAvailableToDraw() && isYes(player)) {
+            player.draw(deck.dealOut());
+            OutputView.printDealOutResult(player);
+        }
+    }
+
+    private boolean isYes(Player player) {
+        String input = InputView.receiveYesOrNoInput(player.getName());
+        return YesOrNo.of(input).isYes();
```

Comment:
```suggestion
        return YesOrNo.isYes(input);
```

와 같이 사용할 수 있다면 사용하기 더욱 수월할 것 같은데 어떻게 생각하시나요?

Reply:
적용했습니다!